556838 -  (typo) Wrong option -CP in snmpnetstat manpage

Source: upstream SVN rev. 18007.

Index: man/snmpnetstat.1.def
===================================================================
--- a/man/snmpnetstat.1.def	(revision 18006)
+++ b/man/snmpnetstat.1.def	(working copy)
@@ -73,7 +73,7 @@
 [common options] [-Ci] [-Cn] [-CI interface] AGENT [interval]
 .br
 .B snmpnetstat
-[common options] [-Ca] [-Cn] [-Cs] [-CP protocol] AGENT
+[common options] [-Ca] [-Cn] [-Cs] [-Cp protocol] AGENT
 .SH DESCRIPTION
 The
 .B snmpnetstat 
@@ -146,7 +146,7 @@
 symbolically).
 This option may be used with any of the display formats.
 .PP
-.BI \-CP " protocol"
+.BI \-Cp " protocol"
 Show statistics about 
 .IR protocol,
 which is either a well-known name for a protocol or an alias for it.  Some
@@ -247,7 +247,7 @@
 .I Single Protocol
 .PP
 When a protocol is specified with the
-.B \-CP
+.B \-Cp
 option, the
 information displayed is similar to that in the
 default display for active sockets, except the
@@ -290,7 +290,7 @@
 Example of using snmpnetstat to show statistics about a specific protocol:
 .PP
 .nf
-% snmpnetstat -v 2c -c public -CP tcp testhost
+% snmpnetstat -v 2c -c public -Cp tcp testhost
 
 Active Internet (tcp) Connections
 Proto Local Address                Foreign Address                 (state)
